verticapy/machine_learning/memmodel/linear_model.py

```diff
@@ -34,6 +34,70 @@ class LinearModel(InMemoryModel):
         ArrayLike of the model's coefficients.
```

Comment:
Perhaps, we can change the InMemoryModel to ``InMemoryModel`` as this is a specific term for our module.

Reply:
done